contributors/design-proposals/storage/local-storage-pv.md

```diff
@@ -634,10 +634,116 @@ limitations outlined above wil be fixed.
 
 #### Block devices and raw partitions
 
-Pods accessing raw block storage is a new alpha feature in 1.8.  Changes are required in
-the Local volume plugin and provisioner to be able to support raw block devices.
+Pods accessing raw block storage is a new alpha feature in 1.9.  Changes are required in
+the Local volume plugin and provisioner to be able to support raw block devices. The local
+volume provisioner will be enhanced to support discovery of block devices and creation of
+PVs corresponding to those block devices. In addition, when a block device based PV is
+released, the local volume provisioner will cleanup the block devices. The cleanup
+mechanism will be configurable and also customizable as no single mechanism covers all use
+cases.
 
-Design TBD
+
+##### Discovery
+
+Much like the current file based PVs, the local volume provisioner will look for block devices
+under designated directories that have been mounted on the provisioner container. Currently, for
+each storage class, the provisioner has a configmap entry that looks like this:
+
+```
+"local-fast":   {
```

Comment:
We changed the configmap to use yaml instead of json

Reply:
Fixed the doc to use yaml format.